metadata-etl/src/main/java/metadata/etl/ownership/DatasetOwnerEtl.java

```diff
@@ -58,7 +59,7 @@ public void extract() throws Exception {
       session =
         jsch.getSession(this.prop.getProperty(Constant.HDFS_REMOTE_USER_KEY), this.prop.getProperty(Constant.HDFS_REMOTE_MACHINE_KEY));
       // use private key instead of username/password
-      session.setConfig("PreferredAuthentications", "publickey");
+      session.setConfig("PreferredAuthentications", "publickey,gssapi-with-mic,keyboard-interactive,password");
```

Comment:
session.setConfig("PreferredAuthentications", "gssapi-with-mic,publickey,keyboard-interactive,password");

I'd prefer putting kerberos first, and then public key.